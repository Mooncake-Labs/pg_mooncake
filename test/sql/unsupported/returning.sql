CREATE TABLE t (a int) USING columnstore;
INSERT INTO t VALUES (1), (2), (3);

-- Test RETURNING clause with UPDATE.
UPDATE t SET a = -a RETURNING a + 1;

-- Test RETURNING clause with DELETE.
DELETE FROM t RETURNING a + 1;

DROP TABLE t;
