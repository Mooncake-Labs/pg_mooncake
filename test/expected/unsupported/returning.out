CREATE TABLE t (a int) USING columnstore;
INSERT INTO t VALUES (1), (2), (3);
-- Test RETURNING clause with UPDATE.
UPDATE t SET a = -a RETURNING a + 1;
ERROR:  RETURNING clause only supports INSERT
-- Test RETURNING clause with DELETE.
DELETE FROM t RETURNING a + 1;
ERROR:  RETURNING clause only supports INSERT
DROP TABLE t;
